reframe/core/pipeline.py

```diff
@@ -421,6 +421,14 @@ class RegressionTest(metaclass=RegressionTestMeta):
     use_multithreading = fields.TypedField('use_multithreading',
                                            bool, type(None))
 
+    #: Specify the maximum time the job can be in the queue before running
+    #:
+    #: Ignored if :class:`None`.
+    #:
+    #: :type: integral or :class:`None`
+    #: :default: :class:`None
+    max_queuing_time = fields.TypedField('max_queuing_time', int, type(None))
```

Comment:
Also why not using a `TimeField` here as in `time_limit`? You do not document also what this integer represents.